meilisearch-http/src/routes/setting.rs

```diff
@@ -12,7 +12,7 @@ use crate::routes::document::IndexUpdateResponse;
 use crate::Data;
 
 #[derive(Default, Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase", deny_unknown_fields)]
```

Comment:
Why did you removed this attribute?

Reply:
Sorry :/ a test related to https://github.com/meilisearch/MeiliSearch/pull/400/commits/4e486d72f1bc2db737fef311cf1065abd19ba828. Doesn't take it into account.